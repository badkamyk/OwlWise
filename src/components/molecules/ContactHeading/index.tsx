import HeadingMain from "../../atoms/HeadingMain";
import MainContentParagraph from "../../atoms/MainContentParagraph";

export default function ContactHeading() {
	return (
		<>
			<HeadingMain firstPhrase={"Contact"} secondPhrase={"us"} />
			<MainContentParagraph
				text={"Got a technical issue? Want to send feedback about a beta feature? Need details about our Business plan? Let us know."} />
		</>
	);
}