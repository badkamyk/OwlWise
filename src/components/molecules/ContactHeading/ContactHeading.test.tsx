import { render, screen } from "@testing-library/react";
import { expect, it } from "vitest";
import ContactHeading from "./index";

it("renders ContactHeading correctly", () => {
	const { container } = render(<ContactHeading />);
	expect(screen.getByText("Contact")).toBeTruthy();
	expect(screen.getByText("us")).toBeTruthy();
	expect(
		screen.getByText(
			"Got a technical issue? Want to send feedback about a beta feature? Need details about our Business plan? Let us know."
		)
	).toBeTruthy();
	expect(container).toMatchSnapshot();
});
