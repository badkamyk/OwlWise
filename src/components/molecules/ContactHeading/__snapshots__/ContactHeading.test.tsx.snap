// Vitest Snapshot v1

exports[`ContactHeading 1`] = `
<div>
  <h2
    class="mb-3 text-3xl font-extrabold text-gray-900 dark:text-white md:text-5xl lg:text-6xl"
  >
    <span
      class="bg-gradient-to-r from-sky-400 to-emerald-600 bg-clip-text text-transparent"
    >
      Contact
    </span>
     
    us
  </h2>
  <p
    class="mb-4 text-gray-900 text-md"
  >
    Got a technical issue? Want to send feedback about a beta feature? Need details about our Business plan? Let us know.
  </p>
</div>
`;
